draft-ietf-webpush-protocol.xml

```diff
@@ -327,6 +338,18 @@ Host: push.example.net
         that these URIs are not disclosed to unauthorized recipients (see <xref
         target="authorization"/>).
       </t>
+      <t>
+        The push service MAY provide a URI for a subscription set resource in a
+        link relation of type "urn:ietf:params:push:set". If provided, the push
+        service supports subscription sets. If available, the user agent SHOULD

```

Comment:
Could this be 'MUST'? Here's my rationale for why. If its should, then I must maintain multiple indexes on every message for a subscription set. One for the subscription URL, and one for the subscription set. At scale, this is extremely expensive. I planned on dumping all notifications to the same subscription set, into a single bucket. If I need to provide access based on the subscription URL separate from the set URL, then I'll need an additional index.